spyder/plugins/ipythonconsole/widgets/main_widget.py

```diff
@@ -46,6 +46,8 @@
 from spyder.widgets.browser import FrameWebView
 from spyder.widgets.findreplace import FindReplace
 from spyder.widgets.tabs import Tabs
+from spyder.utils.workers import WorkerManager
+from spyder.utils.envs import get_list_envs
```

Comment:
Please move these imports a bit above so that they are organized in alphabetical order.